images/builder/Dockerfile

```diff
@@ -123,6 +123,9 @@ COPY --from=builder ["/go/bin/docker-credential-ecr-login", "/usr/local/bin/"]
 # Install docker-compose
 RUN pip install 'docker-compose>=1.24.0,<1.25.0'
 
+# Install dependencies required for testkit
+RUN pip3 install sh pyyaml pycolors
```

Comment:
Could you move this to before the docker-in-docker setup (with a similar stanza?), or after the Snyk setup?